assets/src/apps/delivery/layouts/deck/components/FeedbackContainer.tsx

```diff
@@ -70,7 +70,14 @@ const FeedbackContainer: React.FC<FeedbackContainerProps> = ({
             </button>
           </div>
           <style type="text/css" aria-hidden="true" />
-          <div className="content" style={{ overflow: 'hidden auto !important' }}>
```

Comment:
For some reason, the inline style` style={{ overflow: 'hidden auto !important' }}` was not getting applied so change it to selector base. Starting the selector with '#stage-feedback ' so that this only gets applied to feedback section because there is another '.content' in the main content view.

Reply:
inline styles in React don't work with important, the only way is through text in a style block;